src/System.Text.Json/tests/System.Text.Json.Tests.csproj

```diff
@@ -112,6 +112,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="..\src\System\Text\Json\BitStack.cs" Link="BitStack.cs" />
+    <Compile Include="..\src\System\Text\Json\JsonConstants.cs" Link="JsonConstants.cs" />
```

Comment:
nit: Seems unnecessary to bring in the whole file. The test case could just hard-code the string length to 252 or something like that rather than using the implementation's constant.